Mathlib/Algebra/MonoidAlgebra/Basic.lean

```diff
@@ -63,7 +63,6 @@ def liftMagma [Module k A] [IsScalarTower k A A] [SMulCommClass k A A] :
     { liftAddHom fun x => (smulAddHom k A).flip (f x) with
       toFun := fun a => a.sum fun m t => t • f m
       map_smul' := fun t' a => by
-        -- Porting note (https://github.com/leanprover-community/mathlib4/issues/12129): additional beta reduction needed
```

Comment:
Future idea: should all occurrences of this note simply be deleted? IIRC, that issue almost converged to that conclusion.